### ISSUE TITLE
Customize theme beyond CSS

### ISSUE THREAD

**Issue by maethor** [wekan/wekan#5202]
Hi,

I am currently studying a migration from Trello, and I am ready to put some work on Wekan interface to try to convince my colleagues. I do not want to clone Trello but I would like to compact and reorganize the card details popup. For our usage, too much space is lost with metadata.

Is there a way to alter the jade templates without completely forking wekan? To take a simple example, what if I want to revert https://github.com/wekan/wekan/commit/b33a3ef254719dede124921bde40ce8a47c796fc ?

I know I can do a lot with CSS, but I fear I could be stuck at some point.

Thanks.

**Comment by xet7** [comment]
@maethor 

You can click right sidebar / Board Settings / Card Settings, and there select what fields are visible. No need to edit code.

**Comment by maethor** [comment]
Yes I saw that, and it is a good start, but this is not really what I am asking :) I would like to alter the order of the fields.

**Comment by xet7** [comment]
@maethor

**Comment by maethor** [comment]
OK… The order of fields was just a simple example, my question was broader than that. But I understand. Thank you for your time.

Note that I have already read all the doc, wiki. I have spend hours in PRs, Issues and commit history. I am not asking this question lightly. I just don't know Meteor and Jade so I was wondering if there was a clean way to customize the templates.

So I guess the answer is no. (And no, sorry, I am not able to implement this as a feature because I am not a developer, only a UX enthousiast able to use CSS).

**Comment by xet7** [comment]
@maethor 

For something even more custom, someone asked at #2509 

It is possible to add and remove Custom Fields. There are very many features and settings in WeKan. But not everything is freely customizeable yet.

Problem with custom CSS it, with it, it is very easy to break WeKan UI, like opening something big at top of board. That's why it is not allowed currently. Point is to not allow shoot foot.

https://github.com/wekan/wekan/wiki/Design-Principles

https://github.com/wekan/wekan/wiki/Monkey-Proof-Software
